src/periodic_operators.jl

```diff
@@ -527,6 +527,150 @@ function periodic_derivative_coefficients(source::Holoborodko2008, derivative_or
     PeriodicDerivativeCoefficients(lower_coef, central_coef, upper_coef, mode, derivative_order, accuracy_order, source)
 end
 
+"""
+    LanczosLowNoice()
```

Comment:
Could you please fix the typo "noice" -> "noise" everywhere in the code?

Reply:
Done